src/ErrorHandlerExtensions.cs

```diff
@@ -15,6 +16,31 @@ public static class ErrorHandlerExtensions
         /// <param name="options">Options for handling exceptions and errors</param>
         public static IServiceCollection AddErrorHandler(this IServiceCollection services, Action<ErrorHandlerOptions> options = null)
         {
+            services.Configure<ApiBehaviorOptions>(options =>
+            {
+                foreach(var details in ProblemDetailsDefaults.Defaults)
+                {
+                    // Override existents
+                    if(options.ClientErrorMapping.ContainsKey(details.Key))
+                    {
+                        options.ClientErrorMapping[details.Key].Link = details.Value.Link;
+                        options.ClientErrorMapping[details.Key].Title = details.Value.Title;
+                    }
+                    else
+                    {
+                        // Add news
+                        options.ClientErrorMapping.Add(
+                            details.Key,
+                            new ClientErrorData
```

Comment:
I think we have discussed about this, could we have here a property called like `ProblemDetails`?

Reply:
Here we are adding titles and links to missing status codes